qemu/tests/virtio_fs_support_win_fs.py

```diff
@@ -0,0 +1,78 @@
+import time
+from virttest import error_context, utils_misc
+
+from provider import virtio_fs_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Virtiofs support for file system name specification (windows only).
+
+    1) Before test, backup the image.
+    2) Boot guest with virtiofs device.
+    3) Install viofs driver.
+    4) Create virtiofs service with administrator user.
+    5) Copy a program like 7z-x64.exe and autoit script to the shared dir.
+    6) Create a new user.
+    7) login a common user and install 7z-x64.exe by autoit script.
+    8) After test, restore the image.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    fs_target = params.get("fs_target")
+    package_path = params.get("executable_package_path")
+    autoIt_path = params.get("autoIt_path")
+    script_path = params.get("script_path")
+    reg_add_username = params.get("reg_add_username")
+    reg_add_pwd = params.get("reg_add_pwd")
+
+    vm = env.get_vm(params.get("main_vm"))
+    vm.verify_alive()
+    session = vm.wait_for_login()
+
+    error_context.context("Create the viofs service", test.log.info)
+    virtio_fs_utils.create_viofs_service(test, params, session)
+    error_context.context("add the viofs service", test.log.info)
+    reg_add_cmd = params.get("reg_add_cmd")
+    session.cmd(reg_add_cmd)
+    error_context.context("stop the viofs service", test.log.info)
```

Comment:
Could you change the description with 'restart virtiofs service to make the registry change work', thanks.